diff --git a/Makefile b/Makefile
index 6343ffb..f65eb16 100644
--- a/Makefile
+++ b/Makefile
@@ -6,7 +6,7 @@ BLADERF ?= yes
 CPPFLAGS += -DMODES_DUMP1090_VERSION=\"$(DUMP1090_VERSION)\" -DMODES_DUMP1090_VARIANT=\"dump1090-fa\"

 DIALECT = -std=c11
-CFLAGS += $(DIALECT) -O2 -g -Wall -Werror -W -D_DEFAULT_SOURCE
+CFLAGS += $(DIALECT) -O2 -g -Wall -Werror -W -D_DEFAULT_SOURCE -fcommon
 LIBS = -lpthread -lm -lrt

 ifeq ($(RTLSDR), yes)
